### PR TITLE
conform to theme-check

### PR DESCRIPTION
**Pull request by nuritsha**
use scss, clean-up and document functions.php & theme-parts

### CODE REVIEW

**Comment by Magnethi** [comment]
Hi, can someone please help me out; I'm using the Hello Theme 1.0 on all my domains (www.golfatm.com, www.magnethi.com etc.) and it works perfect. I like the simplicity to total bypass of Wordpress and only using the amazing Elementor :-) 
But I wonder if all the above updates will soon be merged in an updated "official" Hello Theme 1.1 or do I have to update the Theme file manually, and how exactly?

**Comment by marcelturi** [comment]
@Magnethi I see this theme as a starter meaning it's relatively basic so could be used and modified without use of a child theme. However if you're worried about upgrading the base theme easily in the future and also intend to use custom code (PHP, CSS, JS, etc.) then a child theme should be implemented for this purpose so that it's a simple update process.

**Comment by Magnethi** [comment]
Marcelturi, thanks for the reply; I hate child themes, I want everything super simple, the only thing I add to the theme is my GTM code and that's easy to copy/paste into a new theme update. I was more interested in knowing if there will be any official update to the Hello theme or its all geek patch update from in here.

**Comment by marcelturi** [comment]
My guess is that once it passes https://make.wordpress.org/themes/handbook/review/required/ (I patched this theme myself for compliance with the Theme Check plugin a while ago) then the only patches after that will be for compat. with future releases of WP.

File: comments.php [hunk 1]
```diff
@@ -1,4 +1,11 @@
 <?php
+/**
+ * The template for displaying comments
```
Comment (matipojo) [inline comments.php]: period?

File: footer.php [hunk 2]
```diff
@@ -1,4 +1,11 @@
 <?php
+/**
+ * The template for displaying the footer
```
Comment (matipojo) [inline footer.php]: period?

File: functions.php [hunk 3]
```diff
@@ -1,45 +1,90 @@
 <?php
+/**
+ * Elementor Hello Theme functions and definitions
+ *
+ * @link http://elementor.com
```
Comment (matipojo) [inline functions.php]: ?

File: functions.php [hunk 4]
```diff
@@ -1,45 +1,90 @@
...
+/*
+ * Theme Scripts & Styles
+ */
+if ( ! function_exists( 'elementor_hello_theme_scripts_styles' ) ) {
+	function elementor_hello_theme_scripts_styles() {
+		if ( apply_filters( 'elementor_hello_theme_enqueue_style', true ) ) {
+			wp_enqueue_style( 'elementor-hello-theme-style', get_stylesheet_uri() );
+		}
+	}
+	add_action( 'wp_enqueue_scripts', 'elementor_hello_theme_scripts_styles' );
```
Comment (matipojo) [inline functions.php]: move outside

File: functions.php [hunk 5]
```diff
@@ -1,45 +1,90 @@
...
+/*
+ * Register Elementor Locations
+ */
+if ( ! function_exists( 'elementor_hello_theme_register_elementor_locations' ) ) {
+	function elementor_hello_theme_register_elementor_locations( $elementor_theme_manager ) {
+		if ( apply_filters( 'elementor_hello_theme_register_elementor_locations', true ) ) {
+			$elementor_theme_manager->register_all_core_location();
+		}
+	}
+	add_action( 'elementor/theme/register_locations', 'elementor_hello_theme_register_elementor_locations' );
```
Comment (matipojo) [inline functions.php]: move outside

File: package.json [hunk 6]
```diff
@@ -5,13 +5,21 @@
...
+		"grunt-checktextdomain": "^1.0.1",
 		"grunt-contrib-sass": "~1.0.0",
+		"grunt-contrib-watch": "~1.1.0",
 		"grunt-postcss": "~0.9.0",
-		"autoprefixer": "~8.5.2",
-		"cssnano": "~3.10.0",
 		"grunt-sass": "~2.1.0",
+		"grunt-text-replace": "~0.4.0",
+		"grunt-wp-readme-to-markdown-with-extra": "~2.2.0",
+		"load-grunt-tasks": "^4.0.0",
```
Comment (matipojo) [inline package.json]: ?

File: readme.txt [hunk 7]
```diff
@@ -0,0 +1,65 @@
+=== Elementor Hello Theme ===
+
+Contributors: elemntor
+Requires at least: WordPress 4.7
+Tested up to: WordPress 5.0
+Stable tag: 1.0
+Version: 1.0
```
Comment (matipojo) [inline readme.txt]: 1.1.0

File: readme.txt [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+License: GNU General Public License v3 or later
+License URI: https://www.gnu.org/licenses/gpl-3.0.html
+Tags: flexible-header, accessibility-ready, custom-colors, custom-menu, custom-logo, editor-style, featured-images, rtl-language-support, threaded-comments, translation-ready
+
+The theme for Elementor.
```
Comment (matipojo) [inline readme.txt]: The theme for Elementor Plugin

File: readme.txt [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+* Does this theme support any plugins?
+
+Elementor Hello includes support for WooCommerce.
+
+* Can Font Style be added thru the theme's css file?
+
+Best practice is to use the styling capabilities in the Elementor plugin.
+
+== Changelog ==
+
```
Comment (matipojo) [inline readme.txt]: add change log for 1.1.0

File: assets/scss/style.scss [hunk 10]
```diff
@@ -1,17 +1,42 @@
 /*!
 	Theme Name: Elementor Hello Theme
-	Theme URI: https://elementor.com
-	Description: A sample theme for Elementor.
+	Theme URI: https://github.com/pojome/elementor-hello-theme.git
+	Description: The theme for Elementor.
```
Comment (matipojo) [inline assets/scss/style.scss]: The theme for Elementor Plugin

File: package.json (hunk 6 above)
Comment (nuritsha) [reply package.json]: required

**Pull request merged**